server/TypeDBServer.java

```diff
@@ -78,7 +78,7 @@ public class TypeDBServer implements AutoCloseable {
     protected AtomicBoolean isOpen;
     private final CoreConfig config;
 
-    private static TypeDBServer create(CoreConfig config, boolean debug) {
+    static TypeDBServer create(CoreConfig config, boolean debug) {
         configureLogging(new CoreLogback(), config);
         return new TypeDBServer(config, debug, new CoreFactory());
```

Comment:
It's a small smell to me that we're passing in `debug` (boolean) rather than something more granular like `LOG`. See, for example, `LOG.isDebugEnabled() || LOG.isTraceEnabled()` from CoreDatabase.